packages/connect-examples/expo-playground/README.md

```
@@ -0,0 +1,102 @@
+cd hardware-js-sdk/packages/connect-examples/expo-playground
+yarn 
+yarn start
+```
+
+Open `http://localhost:3010` to explore the example.
+
+## 🔗 Resources
+
+- **Main SDK**: [OneKey Hardware SDK](https://github.com/OneKeyHQ/hardware-js-sdk)
+- **OneKey Bridge**: [Download Bridge App](https://help.onekey.so/hc/zh-cn/articles/9740566472335)
+- **API Documentation**: [Hardware API Reference](https://connect.onekey.so/docs) 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)